alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,34 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces)
+Normal:,,
+all elements,/authors,
+"all ""author"" elements",//author,
+"the root element ""authors"" - 2 variants","/* ,  authors, child::authors",
+"the same as 3,  but this time without using the name of the element",/*,
+all elements enclosed by elements  /authors/author,/authors/author,
+"all ""name"" elements (without using names of parent elements) ",//name,
+all elements which have 1 ancestors,,
+"all ""period"" attributes of ""author"" elements - 2 variants","//author[@period] , ",
+"all ""author"" elements with a ""period"" attribute. ",//author[@period],
+"the fourth ""author"" element. ",//author[4],
+"the ""name"" node of the fourth author",//author[4]/name,
+the name of the fourth author (text),//author[4]/name[text()],
```

Comment:
good, but not quite what is expected)
right now it returns name node (as in previous) that has some text, but need to return the text inside this node